### PR TITLE
tests(smoke): remove external resource from dbw

### PR DESCRIPTION
**Pull request by connorjclark**
This will allow us to stop skipping this test in Smokerider.

### CODE REVIEW

File: cli/test/fixtures/dobetterweb/dbw_tester.html
Comment (adamraine) [inline cli/test/fixtures/dobetterweb/dbw_tester.html:471]: nit: why not localhost?

**Review by adamraine — APPROVED** [review]

File: cli/test/fixtures/dobetterweb/dbw_tester.html
Comment (connorjclark) [reply cli/test/fixtures/dobetterweb/dbw_tester.html:471]: Because we explicitly ignore localhost-like urls in the audit. Except we aren't exhaustive so we miss this less common way to specifying localhost :P

**Comment by adamraine** [comment]
Copying my second opinion from chat:

> Yeah on second thought, the fake jquery doesn't add much value to the test because we aren't testing our stack detection on the real thing. If smokerider won't benefit from the jquery stuff either way, then it doesn't make a difference if we split it out.
That way we get the benefit of testing real stack detection (in lighthouse core repo), but can still use the rest of dbw in smokerider
(Assuming stacks is indeed the only check jquery is necessary for)

**Comment by connorjclark** [comment]
There's nothing tricky about detecting jquery - it's as simple as it seems. Using the real library doesn't gain us anything imo.

We have other smokes that test `is-on-http` so to unblock this / avoid the problem with Windows not working, I just removed it from dbw.

**Comment by adamraine** [comment]
I think it's fine to land this PR as is.

That being said, if it's simple enough to include the real jquery library then I think we should. One other option is to check in jquery into the repo as a fixture and use a relative path to include it.

**Pull request merged**
